Sasquatch/Config/Sasquatch.xcconfig

```diff
@@ -15,7 +15,7 @@ SDKROOT = iphoneos
 TARGETED_DEVICE_FAMILY = 1,2
 
 HEADER_SEARCH_PATHS = $(SRCROOT)/../CrashLib/CrashLib
-FRAMEWORK_SEARCH_PATHS = $(PROJECT_DIR)/../AppCenter-SDK-Apple/iOS/**
+FRAMEWORK_SEARCH_PATHS = $(PROJECT_DIR)/../AppCenter-SDK-Apple/iOS/** $(PROJECT_DIR)/../Vendor/iOS/**
```

Comment:
What was this added for?

Reply:
This is a "left over" from the ADAL library usage. Removed it.